### PR TITLE
fix: ⚡️ use unreleased spec keys 🙈

### LINKED ISSUE #137: [bug] NSFW & Photosensitive should not be attributes
### Describe the bug

This is how it looks on UI's that parses attributes, meant more for traits:

image

### Reproduction

_No response_

### Expected behavior

Unclear for now. 
Either don't use these until the spec adopts 'contentRating' or early adopt it.

### Platform and versions

```shell
- macOS
```

### Console output

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by melMass**
This fixes #135 #137

### CODE REVIEW

**Comment by Zir0h** [comment]
So for the indexer, I have to add 2 columns?
accessibility = json object, default {}
contentRating = string, default ""

**Review by LeonNicholls — APPROVED** [review]
Looks good: https://ipfs.io/ipfs/QmPEc3oBxrWWMo26aoEjbP8ukGVxvYFcuYKCKLbM6XJAUz

File: src/pages/mint/index.js [hunk 1]
```diff
@@ -182,19 +182,14 @@ export const Mint = () => {
...
-        {
-          name: 'photosensitiveSeizureWarning',
-          value: photosensitiveSeizureWarning ? 'true' : 'false',
-          type: 'boolean',
-        },
-      ]
+      // Metadata accessibility
+      const accessibility = photosensitiveSeizureWarning
+        ? {
+            hazards: ['flashing'],
```
Comment (melMass) [inline src/pages/mint/index.js:188]: http://kb.daisy.org/publishing/docs/metadata/schema.org/accessibilityHazard.html#value

File: src/pages/mint/index.js [hunk 2]
```diff
@@ -182,19 +182,14 @@ export const Mint = () => {
...
+          }
+        : null
+
+      const contentRating = nsfw ? 'mature' : null
```
Comment (melMass) [inline src/pages/mint/index.js:192]: https://schema.org/contentRating

**Pull request merged**